op-node/node/config.go

```diff
@@ -54,17 +59,25 @@ type Config struct {
 
 	Sync sync.Config
 
-	// [Kroma: START]
-	// To halt when detecting the node does not support a signaled protocol version
-	// change of the given severity (major/minor/patch). Disabled if empty.
-	// RollupHalt string
-	// [Kroma: END]
+	/* [Kroma: START]
+	To halt when detecting the node does not support a signaled protocol version
+	change of the given severity (major/minor/patch). Disabled if empty.
+	RollupHalt string
+	[Kroma: END] */
 
 	// Cancel to request a premature shutdown of the node itself, e.g. when halting. This may be nil.
 	Cancel context.CancelCauseFunc
 
 	// [OPTIONAL] The reth DB path to read receipts from
 	RethDBPath string
+
+	// Conductor is used to determine this node is the leader sequencer.
+	ConductorEnabled    bool
+	ConductorRpc        string
+	ConductorRpcTimeout time.Duration
```

Comment:
Are these used?

Reply:
We don't use it. 
I've chosen to include it in our code and disable it. Additionally, I'll add hidden=true to the flag